src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,90 @@
 import React from 'react';
+import { getUser } from '../../api';
+import { Spinner } from '../Spinner';
+
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  selectedUserId: number;
+  chooseUser: (userId: number) => void;
+}
+
+interface State {
+  id: number;
+  name: string;
+  email: string;
+  phone: string;
+}
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    id: 0,
+    name: '',
+    email: '',
+    phone: '',
+  };
```

Comment:
Если хранить объект целиком, а не каждое свойство, будет проще менеджить стейт
```suggestion
  state: State = {
    user: null;
  };
```